services/ui-src/src/forms/adminDashSelector/adminDashSelector.ts

```diff
@@ -34,7 +34,7 @@ export default {
       props: {
         label: "Select state to view reports:",
         options: dropdownOptions,
-        ariaLabel:
+        "aria-label":
```

Comment:
This has nothing to do with the uploads service being removed, but I noticed an error about this during deployment.